docs/getting-started/linux-installer.md

```diff
@@ -8,6 +8,11 @@ script deploys and configures:
 - Splunk OpenTelemetry Collector for Linux
 - [SignalFx Smart Agent and collectd bundle](https://github.com/signalfx/signalfx-agent/releases) (**x86_64/amd64 platforms only**)
 - [Fluentd (via the TD Agent)](https://www.fluentd.org/)
+  - Optional, **enabled** by default
+  - See the [Fluentd Configuration](#fluentd-configuration) section for more details or how to disable installation
```

Comment:
Should this be "**on** how to disable installation"?

Also, add a period at the end of this sentence.